arangod/Aql/Functions.cpp

```diff
@@ -112,359 +114,399 @@ static_assert(DateSelectionModifier::WEEK < DateSelectionModifier::MONTH,
 static_assert(DateSelectionModifier::MONTH < DateSelectionModifier::YEAR,
               "incorrect date selection order");
 
+typedef void(*format_func_t)(std::string& wrk, tp_sys_clock_ms const&);
+typedef std::unordered_map<std::string,format_func_t> dateMapType;
+std::unordered_map<std::string,format_func_t> dateMap;
+auto unix_epoch = date::sys_seconds{seconds{0}};
 
-/*
```

Comment:
please leave this list in place at the start of the file, or move it else where (like a README) so people starting new aql functions can find it.